app/controllers/offerings_controller.rb

```diff
@@ -0,0 +1,30 @@
+class OfferingsController < ApplicationController
+
+  #show all offers; we don't use this but page exists
+  def index
+    @offers = Offering.all
+  end
+
+  def new
+    @offer = Offering.new
+    @boats = Boat.all
```

Comment:
#show all offers; we don't use this but page exists

anyway, when you add after you added a new offering you will get a list of all boats i guess?